Fix build errors

**Problem description**
The license header format is improper in Process Starter.java, InProcessListenerWithRealStoreAdapter.java, ExternalAdapter.java,OutOfProcessAdapter.java
```
00:31:25 :common:compileJava/var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
00:31:25                                                        ^
00:31:25 /var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
00:31:25                                                         ^
00:31:25 /var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
00:31:25                                                          ^
00:31:25 /var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
00:31:25                                                                  ^
00:31:25 /var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
00:31:25                                                                   ^
00:31:25 /var/jenkins_home/workspace/pravega-branch-builder/common/src/main/java/io/pravega/common/lang/ProcessStarter.java:4: error: unmappable character for encoding ASCII
00:31:25  * Licensed under the Apache License, Version 2.0 (the ???License???);
```

```
:test:integration:compileJava/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                       ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                        ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                         ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                 ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                  ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                   ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                       ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                        ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                         ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                 ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                  ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                   ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                       ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                        ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                         ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                 ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                  ^
/var/jenkins_home/workspace/pravega/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java:4: error: unmappable character for encoding ASCII
 * Licensed under the Apache License, Version 2.0 (the ???License???);
                                                                   ^
```

Javadoc error in ReaderGroupMetrics.java
```
00:32:19 :javadocs/var/jenkins_home/workspace/pravega-branch-builder/client/src/main/java/io/pravega/client/stream/ReaderGroupMetrics.java:21: warning: no @return
00:32:19     long unreadBytes();
00:32:19          ^
```

**Problem location**
Process Starter.java , ReaderGroupMetrics.java, InProcessListenerWithRealStoreAdapter.java and
OutOfProcessAdapter.java

**Suggestions for an improvement**

--- a/client/src/main/java/io/pravega/client/stream/ReaderGroupMetrics.java
+++ b/client/src/main/java/io/pravega/client/stream/ReaderGroupMetrics.java
@@ -17,6 +17,8 @@ public interface ReaderGroupMetrics {
     /**
      * Returns the number of bytes between the last recorded position of the readers in the
      * ReaderGroup and the end of the stream. Note: This value may be somewhat delayed.
+     *
+     * @return The number of unread bytes.
      */
     long unreadBytes();
     

--- a/common/src/main/java/io/pravega/common/lang/ProcessStarter.java
+++ b/common/src/main/java/io/pravega/common/lang/ProcessStarter.java
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- * <p>
- * Licensed under the Apache License, Version 2.0 (the “License”);
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
  */
 package io.pravega.common.lang;
 

--- a/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java
+++ b/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/ExternalAdapter.java
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- * <p>
- * Licensed under the Apache License, Version 2.0 (the “License”);
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
  */
 package io.pravega.test.integration.selftest.adapters;
 

--- a/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java
+++ b/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessListenerWithRealStoreAdapter.java
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- * <p>
- * Licensed under the Apache License, Version 2.0 (the “License”);
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
  */
 package io.pravega.test.integration.selftest.adapters;
 

--- a/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java
+++ b/test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/OutOfProcessAdapter.java
@@ -1,11 +1,11 @@
 /**
  * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- * <p>
- * Licensed under the Apache License, Version 2.0 (the “License”);
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
  */
 package io.pravega.test.integration.selftest.adapters;
 

